Microsoft.Azure.Devices.Edge.sln

```diff
@@ -203,6 +203,8 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ModuleRestarter", "edge-mod
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Relayer", "edge-modules\Relayer\Relayer.csproj", "{C1FDA6CF-7EF6-4C3F-81E7-4C13EE399BF1}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MetricsCollector", "edge-modules\MetricsCollector\MetricsCollector.csproj", "{C01DF161-76F7-47F2-9025-B0D99DB4B814}"
```

Comment:
New project for testing purpose only should move under <repo>/test/modules? is it used for testing?

Reply:
So this might replace the metrics collector currently living in veyalla/metricsCollector. I need to sync with Varun on this though. My only concern is that I have changed the EventHub upload format, so it is not backwards compatible if we decide to deprecate the old module.

Reply:
If it is not for test purpose, this folder should be fine.  Please confirm with Varun.